src/test/java/de/tum/in/www1/artemis/SubmissionExportIntegrationTest.java

```diff
@@ -220,6 +220,15 @@ public void testExportAll_IOException() throws Exception {
         mockedFiles.close();
     }
 
+    @Test
+    @WithMockUser(username = "instructor1", roles = "INSTRUCTOR")
+    public void testExportTextExercise_IOException() throws Exception {
```

Comment:
I think I would change the name to `testExportTextExerciseSubmission_IOException()`, or `testExportTextSubmission_IOException()` as all the service involved in the test is called `textSubmissionExportService`, in any case include _submission_ as this is what is exported?

Reply:
it can be changed like this